README.md

```diff
@@ -98,6 +98,150 @@ You can learn more about it at [wazuh.com][web]
     <img width="640px" src="screenshots/app7.png"/>
 </p>
 
+## Branches
+
+- `stable` corresponds to the latest Wazuh app stable version.
+- `master` branch contains the latest code, be aware of possible bugs on this branch.
+
+## Requisites
+
+- Wazuh HIDS 4.5.1
+- Kibana 7.16.x or 7.17.x
+- Elasticsearch 7.16.x or 7.17.x
+
+## Installation
+
+Ensure that the directory `/usr/share/kibana/data` exists
+If not create it:
+
+```
+mkdir /usr/share/kibana/data
+chown -R kibana:kibana /usr/share/kibana/data
+```
+
+Install the Wazuh app plugin for Kibana
+
+```
+cd /usr/share/kibana
+sudo -u kibana bin/kibana-plugin install https://packages.wazuh.com/4.x/ui/kibana/wazuh_kibana-4.5.1_7.16.0-1.zip
```

Comment:
**nitpick**: this should have into account this pull request https://github.com/wazuh/wazuh-kibana-app/pull/5403/files

Reply:
Solved https://github.com/wazuh/wazuh-kibana-app/pull/5698/commits/85ab302fd365207e6399c2d5d3cbe02b96856774.